mac

```diff
@@ -134,6 +134,18 @@ brew "postgres", restart_service: true
 brew "redis", restart_service: true
 EOF
 
+fancy_echo "Configuring Node ..."
+nvm install stable
+nvm alias default stable
+# shellcheck disable=SC1090
+. "$(brew --prefix nvm)/nvm.sh"
```

Comment:
This needs to come before `nvm install stable` or you won't be able to find the `nvm` command